package com.yash.javatraning.homepractise.ExceptionQ4;

import java.util.Scanner;


public class Test 
{

	public static void main(String[] args) 
	{
		Scanner sc = new Scanner(System.in);
 
	//	Item item =new Item[2];
		System.out.println("===creating array of itme=====");
	//	item[0] = new Item("hair oil", 200, 10);
		/* Department [] arr = new Department[3];
       System.out.println("=====creating array for dept=====");
                    
                     arr[0] = new Department(12344,"Computer Science");
                     arr[1] = new Department(12345,"Hr Department");
                     arr[2] = new Department(12346,"Facility team");
       System.out.println("department data  in department arr 0:");
       arr[0].display();
		 * */
		
		
	}

}
